backend/baseclass.pm

```diff
@@ -855,7 +855,11 @@ sub proxy_console_call {
         # Move the decision to actually die to the server side instead.
         # For this ignore backend::baseclass::die_handler.
         local $SIG{__DIE__} = 'DEFAULT';
-        $wrapped_result->{result} = $console->$function(@$args);
+        if ($wrapped_call->{wantarray}) {
+            $wrapped_result->{result} = [$console->$function(@$args)];
```

Comment:
as codecov is reporting that body of the if is now not covered by tests. Please add according test coverage.

Reply:
yes, we should have a test that actually uses the list context.

Reply:
done